### PR TITLE
Launch correct version of Axon

### PR DESCRIPTION
**Pull request by GrantPerkins**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "format": "prettier --write .",
     "test:format": "prettier --check .",
     "test:lint": "yarn workspaces run test:lint",
-    "dockerdev": "docker build --build-arg AXON_VERSION=edge -t axon:local . && docker run -t -i -v /var/run/docker.sock:/var/run/docker.sock -v wpilib-axon-volume:/usr/src/app/packages/server/data -p 3000:3000 -p 4000:4000 axon:local"
+    "dockerdev": "docker build --build-arg AXON_VERSION=0.0.1 -t axon:local . && docker run -t -i -v /var/run/docker.sock:/var/run/docker.sock -v wpilib-axon-volume:/usr/src/app/packages/server/data -p 3000:3000 -p 4000:4000 axon:local"
```
Comment (AustinShalit) [inline package.json]: This means we will be dev testing a released version. I think we want to replace this with edge or local (and build the containers locally too).

File: packages/server/src/mL/index.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ export default class MLService {
    * pull docker images.
    */
   async initialize(): Promise<void> {
+    console.log(process.env);
```
Comment (AustinShalit) [inline packages/server/src/mL/index.ts]: I think we should revert this line.

File: package.json (hunk 1 above)
Comment (GrantPerkins) [reply package.json]: oops, that is an artifact from debugging

File: packages/server/src/mL/index.ts (hunk 2 above)
Comment (GrantPerkins) [reply packages/server/src/mL/index.ts]: another artifact.

**Review by AustinShalit — APPROVED** [review]

**Pull request merged**